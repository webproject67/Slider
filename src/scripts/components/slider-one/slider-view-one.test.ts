import SliderModel from '../model/slider-model';
import SliderViewOne from './slider-view-one';

const model = new SliderModel();
const slider = new SliderViewOne(model);

test('spyOn mouseDown toggle one', () => {
  const somethingSpy = jest.spyOn(slider, 'toggleMouseDown');
  let evt: any;
  slider.toggleMouseDown(evt);
  expect(somethingSpy).toHaveBeenCalledTimes(1)
});

it('jest snapshots', () => {
  expect(slider.element).toMatchSnapshot()
})