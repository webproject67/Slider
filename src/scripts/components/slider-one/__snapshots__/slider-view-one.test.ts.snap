// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest snapshots 1`] = `
<div
  class="slider__wrapper"
>
  
      
  <div
    class="slider__inner"
    data-testid="slider-one"
  >
    
        
    <div
      class="slider__scale"
    >
      
          
      <div
        class="slider__bar"
        style="margin-right:0%"
      />
      
        
    </div>
    
        
    <div
      class="slider__toggle slider__toggle_maximum"
      style="left:100%"
    />
    
      
  </div>
  
    
</div>
`;
